src/__tests__/applicationForms.int.test.ts

```diff
@@ -135,7 +119,6 @@ describe('/applicationForms', () => {
       `);
 			const result = await agent
 				.get('/applicationForms/2')
-				.query({ includeFields: 'true' })
```

Comment:
Does it always include fields now? I assume so seeing the type below.

Reply:
It does!

Reply:
Oh, and it's right there in your commit message too, ha:
> Removes the query parameter for optionally returning deep / shallow ApplicationForms; the API now always returns deep ApplicationForms.